tensorflow_federated/python/core/impl/executors/federating_executor.py

```diff
@@ -35,6 +35,9 @@
 from tensorflow_federated.python.core.impl.executors import executor_utils
 from tensorflow_federated.python.core.impl.executors import executor_value_base
 
+import tf_encrypted as tfe
+from tf_encrypted.primitives.sodium.python import easy_box
```

Comment:
you should be able to simply use `from tf_encrypted.primitives.sodium import easy_box`, ie without `python`

Reply:
good point!